ivy/data_classes/container/experimental/statistical.py

```diff
@@ -444,6 +444,130 @@ def nanmean(
             self, axis=axis, keepdims=keepdims, dtype=dtype, out=out
         )
 
+    @staticmethod
+    def static_nanmin(
```

Comment:
I think it's better if we make separate PRs for separate functions! 😄 Let's keep this one only for `nanmin`

Reply:
It is for same function nanmin. If you check the file you will see for every function there is a static method created as mine.

Reply:
Sounds good!